Makefile

```diff
@@ -151,3 +164,7 @@ bluetoothdevice_image:
 .PHONY: bluetoothdevice_lint
 bluetoothdevice_lint:
 	make -C mappers/bluetooth_mapper lint
+
+install: ${exes}
+	install -d ${INSTALL_DIR}/bin/
+	install $^ ${INSTALL_DIR}/bin/
```

Comment:
if we add `install` rule, We need to consider the installation path of the configuration file.